src/components/tab-title/tab-title.scss

```diff
@@ -190,11 +192,21 @@
   }
 }
 
+:host([closable]) {
+  .content {
+    padding-inline-start: 1.5rem;
```

Comment:
@alisonailea @Elijbet are there any tokens we can use as values for these paddings?f

Reply:
If this padding should be referenced from `theme()`/tailwind we could do it as a followup.

Reply:
Let me see what design says about the centering approach in general and yeah that would be the next question.